scalyr_agent/builtin_monitors/journald_monitor.py

```diff
@@ -557,16 +557,12 @@ def format_msg(
     ):
         string_buffer = six.StringIO()
 
-        string_buffer.write(
-            "%s %s" % (metric_name, scalyr_util.json_encode(metric_value))
-        )
+        string_buffer.write("%s %s" % (metric_name, metric_value))
 
         if extra_fields is not None:
             for field_name in extra_fields:
                 field_value = extra_fields[field_name]
-                string_buffer.write(
-                    " %s=%s" % (field_name, scalyr_util.json_encode(field_value))
-                )
+                string_buffer.write(" %s=%s" % (field_name, field_value))
```

Comment:
For this, I think we might want to keep this as json_encode, but surround it with quotes.  For example,

```
" %s=\"%s\"" % (field_name, scalyr_util.json_encode(field_value))
```

I think we should do it that way because it simplifies customers having to write parsers to extract the extra fields values.  In your test cases below, you have an example where the output would be `timestamp=2015-09-05 13:17:04.944355`.  Because of the space, it's hard to tell where that key value pair ends.  Instead, with the change I'm proposing, it would be `timestamp="2015-09-05 13:17:04.944355"`.

Reply:
`json_encode` already adds quotes around it, so I think we might not need any change at all here.